src/util/SchemaUtil.cpp

```diff
@@ -217,6 +245,11 @@ StatusOr<DataSet> SchemaUtil::toShowCreateSchema(bool isTag,
     } else {
         createStr += "\"\"";
     }
+    if (prop.__isset.comment) {
+        createStr += ", comment = \"";
```

Comment:
upper case

Reply:
It's same with properties like ttl_col, ttl_duration.